### PR TITLE
Add missing title for day 22

### PR DESCRIPTION
**Pull request by mhmdanas**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -349,6 +349,8 @@ But From dart 2.3 and above, we can use Spread Operator (`...`) to add collectio
 
 [try in dartpad](https://dartpad.dev/98c2ab9d41fb2c20cc67c94956972721)
 
+## #Day22 Make a class callable like a function
```
Comment (erluxman) [inline README.md:352]: Can you just make it a "Callable class"?

Comment (mhmdanas) [reply README.md:352]: Will do!

Comment (mhmdanas) [reply README.md]: Done!

**Comment by erluxman** [comment]
Thanks man..

**Pull request merged**